server/neural_network_interface.py

```diff
@@ -77,6 +82,21 @@ def _build_flip(self, sid, block_id):
             'move_number': self._movement_count_dict[sid]
         }
 
+    def _build_ambiguous(self, letter, color, block_candidates):
+        return {
+            'type': 'ambiguous',
+            'predicted_letter': letter if letter != 'None' else False,
+            'predicted_color': color if color != 'None' else False,
```

Comment:
What happens if we pass `None` instead of `False`? It makes more sense logically, but I'm not sure how that would be received on the JavaScript side.

Reply:
Nothing exciting happens, I would just have to check for `'None'` instead of false, but I disagree that it makes more sense logically. Passing a falsey value to indicate that there is no color seems more natural to me.

Reply:
I was thinking the keyword `None` rather than the String value `'None'`, unless JavaScript sees it on the other end as the string `'None'`, in which case yes that doesn't really improve anything. I'd hope it would come out the other end as `undefined`.

Reply:
My guess is that it would be `'None'`, but I wouldn't really like `undefined` either because I think that it is just meant for undefined variables.